### PR TITLE
Fix pkg_rpm rules for bazel 0.13+

### PR DESCRIPTION
**Pull request by ixdy**
**What this PR does / why we need it**: next step in addressing https://github.com/kubernetes/kubernetes/issues/63108; we can use the substitutions supported by bazel 0.13+ to get the `pkg_rpm` rules to work properly again.

I've also added a filegroup to allow building all of the RPMs easily with `bazel build //build/rpms`. Note that since these are manual, `bazel build //...` will still skip building them. 

**Release note**:

```release-note
NONE
```

/assign @BenTheElder 
cc @sigma

### CODE REVIEW

**Review by BenTheElder — APPROVED** [review]
/lgtm
/hold

**Comment by sigma** [comment]
can/should the bazel version requirement be encoded somewhere?

**Comment by ixdy** [comment]
@sigma it is, in https://github.com/kubernetes/kubernetes/blob/d39ff8e651b7b56e598cfe78f3a64b2d04cd671e/build/root/WORKSPACE#L42.

I intend to bump that as soon as we're using bazel 0.13 on prow.
/retest

**Comment by ixdy** [comment]
alternately we can wait to merge this PR until prow is using a newer bazel.

@BenTheElder have we updated the canary job yet?

**Comment by BenTheElder** [comment]
> @BenTheElder have we updated the canary job yet?

Nope, that slipped. Doing that now.

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/test-infra/pull/7921

**Comment by ixdy** [comment]
the bazel build tests aren't likely to be very relevant here, since we no longer build these RPMs in CI.

**Comment by ixdy** [comment]
/hold cancel
Can I get a re-lgtm please?

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Pull request merged**
